main.py

```diff
@@ -39,12 +39,13 @@ def get_players():
         players = [(data['members'][member]['name'],
                     data['members'][member]['local_score'],
                     data['members'][member]['stars'],
-                    int(data['members'][member]['last_star_ts'])) for member in data['members']]
+                    int(data['members'][member]['last_star_ts']),
+                    data['members'][member]['completion_day_level']) for member in data['members']] ##This is the data for all the days
```

Comment:
Could we avoid inline comments if possible? I think this comment would work well above the `players =` line

Reply:
And the missing space after the '#'

Reply:
I've taken it out you already have a comment above which explains whats happening